apps/controllerx/cx_core/integration/tasmota.py

```diff
@@ -0,0 +1,56 @@
+import json
+from typing import Any, Dict, Optional
+
+from appdaemon.plugins.mqtt.mqttapi import Mqtt
+from cx_const import DefaultActionsMapping
+from cx_core.integration import EventData, Integration
+
+
+class TasmotaIntegration(Integration):
+    name = "tasmota"
+
```

Comment:
Since `device` attribute is mandatory, I suggest overwriting the `__init__()` to check if kwargs contains `device`.

Reply:
Regarding this, I will need some help, as this is the first time I code python... :-\